tests/functional/test_tracing.py

```diff
@@ -13,6 +14,38 @@
 mountpoint = "/mnt"
 
 
+def test_iotracer_multiple_instances():
+    """
+        title: Check if it’s possible to run concurrent tracing processes.
+        description: |
```

Comment:
It's in case when description has more than one line. It can be deleted here, but I left it to remember that multi-line comment needs this or in case of copy-paste.